supervisely/api/issues_api.py

```diff
@@ -0,0 +1,481 @@
+            ApiField.UPDATED_AT,
+            ApiField.DATASET_ID,
+            ApiField.PROJECT_ID,
+            ApiField.IMAGE_NAME,
+            ApiField.NAME,
+        ]
+
+    @staticmethod
+    def info_tuple_name():
+        """Name of the tuple that represents IssueInfo."""
+        return "IssueInfo"
+
+    def get_list(self, team_id: int, filters: List[Dict[str, str]] = None) -> List[IssueInfo]:
+        """Get list of issues in the specified team.
+
+        :param team_id: Team ID.
+        :type team_id: int
+        :param filters: List of filters to apply to the list of issues.
+        :type filters: List[Dict[str, str]], optional
+
+        :return: List of issues.
+        :rtype: List[IssueInfo]
+
+        :Usage example:
+
+        .. code-block:: python
+
+            import os
+            from dotenv import load_dotenv
+
+            import supervisely as sly
+
+            # Load secrets and create API object from .env file (recommended)
+            # Learn more here: https://developer.supervisely.com/getting-started/basics-of-authentication
+
+            if sly.is_development():
+                load_dotenv(os.path.expanduser("~/supervisely.env"))
+
+            api = sly.Api.from_env()
+
+            # Get list of issues in specified team.
+            issues = api.issues.get_list(team_id=1)
+        """
+        return self.get_list_all_pages(
+            "issues.list",
+            {ApiField.FILTER: filters or [], ApiField.TEAM_ID: team_id},
+            ensure_paginations=False,  # ! WARNING! Remove this parameter after bugfix on the API!
+        )
+
+    def get_info_by_id(self, id: int) -> IssueInfo:
+        """Get information about the issue by its ID.
+
+        :param id: Issue ID.
+        :type id: int
+        :return: Information about the issue.
+        :rtype: IssueInfo
+
+        :Usage example:
+
+        .. code-block:: python
+
+            import os
+            from dotenv import load_dotenv
+
+            import supervisely as sly
+
+            # Load secrets and create API object from .env file (recommended)
+            # Learn more here: https://developer.supervisely.com/getting-started/basics-of-authentication
+            if sly.is_development():
+               load_dotenv(os.path.expanduser("~/supervisely.env"))
+
+            api = sly.Api.from_env()
+
+            # Get information about the issue by its ID.
+            issue_info = api.issues.get_info_by_id(1)"""
+        response = self._get_response_by_id(id, "issues.info", id_field=ApiField.ID)
+        return (
+            self._convert_json_info(response.json(), skip_missing=True)
+            if (response is not None)
+            else None
+        )
+
+    def add(
+        self,
+        team_id: int,
+        issue_name: str,
+        comment: Optional[str] = None,
+        assignees: Optional[List[int]] = None,
+        is_local: bool = False,
+    ) -> IssueInfo:
+        response = self._api.post(
```

Comment:
dosctring ?
`is_local` means visible only for your team?
If yes, I would propose to use `team_scope_only`